### PR TITLE
(doc): migrate to eta from ejs

### PR DESCRIPTION
**Pull request by tamil777selvan**
## Proposed changes

Fix - #6640

- migrate to eta from markdox & ejs

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Comment by christian-bromann** [comment]
@tamil777selvan thanks for taking a stab at this. I am not sure if mustache is a good choice. It hasn't seen any release for more than 2 years and seems like abandoned to me. Any chance we can use [`eta`](https://www.npmjs.com/package/eta) which is one I've been using in the past and very liked.

**Comment by tamil777selvan** [comment]
Hi @christian-bromann, I followed your advice and started using ETA. Could you please take a look at it whenever you get a chance? Thank you!

File: scripts/templates/template.eta [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+<% } %>
+
+<%= it.description %>
+
+<% if (it.alternativeCommands && it.alternativeCommands.length) { %>
+:::info
+
+<% /* This protocol command is embedded in the following convenient method<%= it.alternativeCommands.length > 1 ? 's' : '' %>: <% for (const altCommand of it.alternativeCommands) { %>[<%= altCommand.split('/').pop() %>](/docs/api/<%= altCommand %>)<% } %>. It is recommended to use <%= it.alternativeCommands.length > 1 ? 'these commands' : 'this command' %> instead. */ %>
+
+This protocol command is embedded in the following convenient method<%= it.alternativeCommands.length > 1 ? 's' : '' %>: <%= it.alternativeCommands.map((command) => `[${command.split('/').pop()}}](/docs/api/${command})`).join(', ') %>. It is recommended to use <%= it.alternativeCommands.length > 1 ? 'these commands' : 'this command' %> instead.
```
Comment (christian-bromann) [inline scripts/templates/template.eta:21]:
```suggestion
This protocol command is embedded in the following convenient method<%= it.alternativeCommands.length > 1 ? 's' : '' %>: <%= it.alternativeCommands.map((command) => `[\`${command.split('/').pop()}\`](/docs/api/${command})`).join(', ') %>. It is recommended to use <%= it.alternativeCommands.length > 1 ? 'these commands' : 'this command' %> instead.
```

**Review by christian-bromann — COMMENTED** [review]
Can we also remove the `postinstall:vulnerabilities` command and the file connected to it?

The rest looks good to me, great job!

**Comment by tamil777selvan** [comment]
> Can we also remove the `postinstall:vulnerabilities` command and the file connected to it?

I have updated it @christian-bromann, please have a look. Thanks!

**Review by christian-bromann — APPROVED** [review]
Awesome, LGTM 👍

**Pull request merged**